src/infrasys/cost_curves.py

```diff
@@ -45,15 +43,13 @@ class FuelCurve(ProductionVariableCostCurve):
             description="The underlying `ValueCurve` representation of this `ProductionVariableCostCurve`"
         ),
     ]
-    vom_units: Annotated[
+    vom_cost: Annotated[
         InputOutputCurve,
         Field(description="(default: natural units (MW)) The units for the x-axis of the curve"),
-    ] = InputOutputCurve(
-        function_data=LinearFunctionData(proportional_term=0.0, constant_term=0.0)
-    )
+    ] = LinearCurve(0)
     fuel_cost: Annotated[
-        float,
+        pint.Quantity | float | None,
```

Comment:
When will this be None? You default to 0. Will a user actually pass None?

Reply:
I think we can pint.Quantity or float and leave out the None.